queries/progress.sql

```diff
@@ -1,5 +1,9 @@
 -- name: getProgressForProfile :many
-SELECT p.episode_id, p.show_id, p.progress, p.duration, COALESCE((p.progress::float / NULLIF(p.duration,0)) > 0.8, false)::bool AS watched
+SELECT p.episode_id,
+       p.show_id,
+       p.progress,
+       p.duration,
+       COALESCE((p.progress::float / NULLIF(p.duration, 0)) > 0.8, false)::bool AS watched
```

Comment:
How does SQL behave in division with 0?

Reply:
It ends up as null.

But this will be changed in the other PR (with incremental watched instead of calculated on the fly)